lint-staged.config.js

```diff
@@ -0,0 +1,13 @@
+export default {
```

Comment:
I personally do not like lint-staged, it requires new commands run here instead of the ones in package.json and for me it seems to not always work correctly. I know this is in the extension I would prefer we just lint all outstanding changes. Do you have any thoughts here @reneaaron @bumi ?

Reply:
decision is to leave as is. Lint-staged runs faster and we can run full lint on CI